modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -1,42 +1,142 @@
-When you are ready to migrate from 6.5.4 to a 7.x.y version, make sure you migrate to at least 7.1.1 to be sure that these fixes are present in your new version after migration.
-To migrate from 6.5.4 to 7.1.1 or later there are two phases: first you upgrade to 7.0.0 then to the latest version.
-You are recommended not to start 7.0.0 after you migrate to it, but to proceed immediately with the second phase of the migration.
+.*Special attention required on:*
+* Several bugs affect legacy Bonita versions by preventing a smooth Bonita Runtime update with BPM services paused when:
+** The source version is older than Bonita 7.8.0, and the target version is between Bonita 7.8.0 and Bonita 7.11.5
+** The source version is older than Bonita 7.10.5 and the target version is older than Bonita 2021.1
+* Deployed process definitions: The processes will continue to run using the definition created in the previous Bonita version. 
 ====
 
-*JRE requirements:*
+.Before launching the update script, there are *manual activities* to be done, like: 
+* Go through the release notes of all the versions till the one targeted for the update, for changes/deprecations/removals and see if they impact your existing developments.
+____
+*For example:* when updating from 7.9.y to 7.10.0, you will need to read all the release notes in between those versions.
+____
+
+// link to special cases and troubleshouting to be added => 
+* Go through the [Special cases] and [Troubleshooting] paragraphs below.
+
+* Runtime custom configuration. You will need to re-apply the customizations after the update process finishes. *If the target version of the update is Bonita 7.3 or greater, you can use xref:runtime:bonita-platform-setup.adoc[the setup tool] to push the configuration files.*
+
+* Process definition sources (.bos files) require a manual update. First export then import them into the new Bonita Studio version.
+* Custom connectors, actor filers, data types, custom pages, RestAPI extensions: will for sure continue to work in the new version if no custom code has been added, default Bonita code. If you did add custom code, before starting the update, check in the release notes for any breaking changes and test them thoroughly after the update.
+* Check that the database version is up-to-date and supported by the new Bonita version.
+* Secure the update by doing backups. Better safe than sorry.
+
+.Following elements will be updated *automatically* when launching the update script:
+* Engine server
+* Engine database (including all data on active and archived process instances) 
+* Organization definition
+* Log files from the previous versions will not be touched. A new dedicated folder for the target version will be created to store the log files. 
+* Runtime data (in the database after Bonita Runtime 7.3)
+* Configuration files (in the database after Bonita Runtime 7.3) will be replaced with the default configuration files of the new version
```

Comment:
@pabloalonso is it true?

Reply:
For some configuration files that cannot be modified by the customer, this is true we replace them.
For specific files, ~migration~ update steps handle the change, so that the new version is updated, without breaking the customizations.